### ISSUE TITLE
[Request] w3m

### ISSUE THREAD

**Issue by indrakaw** [lukesampson/scoop#3353]

**Comment by 0x326** [comment]
@indrakaw How is `w3m` packaged? (e.g. How would you go about installing it without Scoop? Is it in a zip file?  Is it an installer?  Can the installer be extracted using 7zip?)

**Comment by indrakaw** [comment]
Normally, I install it via MSYS2 on Windows. I wonder, is it has Windows binnary or not.

**Comment by 0x326** [comment]
If it depends on MSYS2, Cygwin, or an alternative compatibility layer, it's probably out of scope of Scoop and better handled by the layer's respective package manager.

**Comment by rasa** [comment]
This issue belongs in the [Main](https://github.com/ScoopInstaller/Main) bucket. Unfortunately, we cannot move this issue on this end, so it is being closed.
If this is still an issue, please create a new issue in the Main bucket via https://github.com/ScoopInstaller/Main/issues/new?body=Moved+from+https%3A%2F%2Fgithub.com%2Flukesampson%2Fscoop%2Fissues%2F3353
